cypress/integration/LandingPage.feature

```diff
@@ -3,11 +3,20 @@ Feature: Landing Page
   I want to see useful things on the landing page
   So I know what to do next
 
-  Scenario: Landing page shows correct heading
+  Background:
     When I visit the "landing" page
+
+  Scenario: Landing page shows correct heading
     Then I see "Find missing family" in the page heading
 
   Scenario: Landing page has make enquiry button
-    When I visit the "landing" page
-    And I see a "make an enquiry" button linking to the "make an enquiry" page in the "hero" element
+    Then I see a "make an enquiry" button linking to the "make an enquiry" page in the "hero" element
     And I see a "make an enquiry" button linking to the "make an enquiry" page in the "content" element
+
```

Comment:
We could probably also argue here that the button existing is covered by the it working scenario below.